docs/get-started/allocation/adding-a-new-allocation.md

```diff
@@ -0,0 +1,20 @@
+# Adding a new Resource Allocation to the project
+
+If one resource allocation is not sufficient for a project, PI or project managers
+can request for another allocation(s) by clicking on the "Request Resource Allocation"
+button on the Allocations section of the project details. This will show the page
+where all existing users for the project will be listed on the bottom of the request
+form. PIs can select all or only desired user(s) to request the resource allocations
```

Comment:
form. PIs can select desired user(s)

Reply:
you don't need "to request the resource allocations" you already told them you are talking about resource allocations.

Reply:
Please don't mark items as resolved unless you do exactly what was requested so that it can be reviewed by the reviewer.